examples/seismic/source.py

```diff
@@ -108,7 +108,7 @@ def show(self, idx=0, time=None, wavelet=None):
         plt.figure()
         plt.plot(time, wavelet)
         plt.xlabel('Time (ms)')
-        plt.ylabel('Velocity (km/s)')
+        plt.ylabel('Amplitude')
```

Comment:
No unit?

Reply:
Would depend on the kernel, acoustic would be pressure, but elastic wouldn't. Better left without any in my opinion.

Reply:
ok, fair enough. Good to go then.